public/app/core/components/Layers/EditLayerName.tsx

```diff
@@ -0,0 +1,159 @@
+import { css, cx } from '@emotion/css';
```

Comment:
Is it possible to use the current `LayerName` component here (so we can have the same UX (with edit button appearing right by the element name vs as a separate option (potentially a little confusing as not clear if edit is for element vs just name)

Also if we can match the cursor type behavior to the original layer list that would be great :) (grab hand vs pointer on actionable buttons)

https://user-images.githubusercontent.com/22381771/180824860-05d9ff6a-ace5-4661-ad78-f33232398aab.mov

Reply:
Ok, I added it back - I've only made a separate component because it was a bit annoying when I tested it out, as it was getting into edit mode a lot more than I wanted. But it seems ok.